.mockery.yaml

```diff
@@ -0,0 +1,10 @@
+with-expecter: True
+dir: mocks/{{.InterfaceDirRelative}}
+mockname: "{{.InterfaceName}}Mock"
+outpkg: "{{.PackageName}}"
```

Comment:
By default, this option is already set to `{{.PackageName}}`.
```suggestion
```

Reply:
updated